### ISSUE TITLE
Allow user to gamefully obtain my resume

### ISSUE THREAD

**Issue by twistedstream** [twistedstream/twistedstream-web-legacy#1]
General workflow:
- User logs in with a deterministically calculable username and password
- The website returns a JWT token and instructions
- The user then must use curl (or similar tool) to perform a GET with the JWT token (via the Authorization header) to obtain links to the actual resume and Careers 2.0 page

Tasks:
- [x] Question API resource that returns a JWT token if the caller successfully answers the question
- [x] Resume API resource that takes the JWT token in the `Authorization` request header
- [ ] A frontend UI component that prompts the user for an answer and calls the Question resource
- [ ] A related frontend UI component that reports back the JWT token and provides instructions on how to access the Resume resource

**Comment by twistedstream** [comment]
To gain access, the user needs to complete this JavaScript function to produce the sum of the two inputs, with the result in binary format:

``` js
function sillyStupidAddingMachine(dec, hex) {
  // dec will be a decimal integer number
  // hex will be a hexadecimal string
  // return the sum of dec and hex, but in binary format
  [ their code here ]
}
```

One solution might be:

``` js
var a = parseInt(dec);
var b = parseInt(hex, 16);
var c = a + b;
return c.toString(2);
```

**Comment by twistedstream** [comment]
Resources:

## Question

```
POST /api/question
```

``` json
{
  "code": "..."
}
```

Response:
`200 OK`

``` json
{
  "access_token": "JWT_TOKEN"
}
```

## Resume

```
GET /api/resume
Authorization: Bearer JWT_TOKEN
```

Response:
`200 OK`

``` json
{
  "resume_url": "RESUME_URL",
  "stack_overflow_careers_url": "STACK_OVERFLOW_CAREERS_URL"
}
```

**Comment by twistedstream** [comment]
Consider converting the **Resume** resource from a resource of links to two separate resources, a personal and a Stack Overflow:

## Personal Resume

```
GET /api/personal_resume
Authorization: Bearer JWT_TOKEN
```

The endpoint could support multiple media types passed in the `Accept` request header:
- PDF (`application/pdf`) **default**
- HTML (`text/html`)
- Word Doc (`application/vnd.openxmlformats-officedocument.wordprocessingml.document`)
- Text (`text/plain`)

All of which can be rendered via the backing Google document.

## StackOverflow Resume

```
GET /api/stackoverflow_resume
Authorization: Bearer JWT_TOKEN
```

The response will be a redirect to the StackOverflow Careers resume:

```
302 Found
Location: http://careers.stackoverflow.com/my-resume
```

**Comment by twistedstream** [comment]
Implemented at 3ec26a74d17824be9ed1f06414183479d3564166
